docs/localizability.md

```diff
@@ -24,9 +24,11 @@ Unfortunately, NuGet.Client does not use OneLocBuild, and any resource comments
 Although the steps below are for the dev branch, the process is similar for release branches.
 
 1. A PR merges into the dev branch.
-1. A NuGet.Client build generates a `localizationArtifacts` artifact.  This artifact contains newly built (but unsigned) assemblies and .lcg files.  The .lcg files contain information extracted from .resx files and are the primary input for later localization.
+1. A NuGet.Client build generates a `localizationArtifacts` artifact.  This artifact contains newly built Authenticode-signed assemblies [(see PR#6170)][12]) containing English resources and .lcg files.
```

Comment:
```suggestion
1. A NuGet.Client build generates a `localizationArtifacts` artifact.  This artifact contains newly built assemblies containing English resources and .lcg files.
```

If someone needs to investigate the loc process in a year or more from now, I'm not sure why the PR would be interesting to call out specifically.  In fact, the fact that it's signed or not also doesn't seem important to me.

The loc team's tools don't have a requirement about signed or unsigned assemblies.

Reply:
> The loc team's tools don't have a requirement about signed or unsigned assemblies.

This documentation is for the NuGet Client team, not the loc team.

> In fact, the fact that it's signed or not also doesn't seem important to me.

The original document indicated that these were unsigned assemblies. This is an update reflecting the new state of these assemblies.
This change is part of the SFI work to ensure we sign assemblies. If localized assemblies are not signed, we are no longer meeting SFI requirements.